### ISSUE TITLE
2 мелкие орфографические прпвки

### ISSUE THREAD

**Issue by dinovasya** [tretyakovsa/Sonoff_WiFi_switch#2]
Воскресенья","Понидельник

Замените на Воскресенье","Понедельник

Спасибо

**Comment by renat2985** [comment]
Спасибо, исправил. Буду рад если найдете еще какие-то недочеты.

**Comment by dinovasya** [comment]
Да, в твиттер отписал. Короче сразу при первом запуске sonoff. На первой
странице setup.htm ввести свою сетку вай-фай невозможно. Тип поля
неправильный, не текстовый, а выпадающий список. А в нём пусто.  Прилагаю
скрин.


17 дек. 2016 г. 10:43 пользователь "Renat" <notifications@github.com>
написал:

**Comment by dinovasya** [comment]
https://postimg.org/image/5gf9apb0p/
https://postimg.org/image/rlk73oisx/

**Comment by renat2985** [comment]
Хм. Должно быть так:
![_101](https://cloud.githubusercontent.com/assets/1230815/21285366/f3211c5c-c43e-11e6-9257-c7af63a297a0.png)

То есть в выпадающем списке вы должны видеть все доступные Wi-Fi сети. Я предполагаю вам нужно обновить файловую систему и сделать рефрешь на странице setup.htm, т.к. в последних версиях кешируются js файлы.

Если не получится, свяжитесь со мной в skype: renat2985 
Буду на связе через 2 часа.

**Comment by dinovasya** [comment]
я каждый раз файловую перешиваю.. но вот кеширование возможно. Хром собака
тот еще- куки удалять не помогает- в нем только сброс до заводских настроек
помогает кеш снести.
Ну вот этот выпадающий список пуст.

17 декабря 2016 г., 10:58 пользователь Renat <notifications@github.com>
написал:

**Comment by dinovasya** [comment]
да, дело было в хроме. сбросил настройки до заводских- сраболтало. в хроме теперь открылось нормально..

**Comment by dinovasya** [comment]
еще одна орфографическая правка:
Здесь указается время в секундах как долго будет работать реле.
Здесь указЫВается время в секундах как долго будет работать реле.

А лучше кратко написать: Продолжительность работы таймера.

Кстати данный блок на всех языках выглядит на Русском ))

**Comment by renat2985** [comment]
Ошибку исправил. Спасибо.

**Comment by jkjk007** [comment]
Гостиная пишется с одним "Н".

**Comment by renat2985** [comment]
@jkjk007 я не русский мне можно и с двумя :) это местонахождение, там вы можете написать все что угодно.
